gravityflow/gravityflowchecklists
# Fix checklist not completing

This PR:
1. Fixes an issue where the checklist does not display as complete if one of the forms is marked as exempt.
2. Changes the checklist completion logic is to display as complete only when the workflow of the final form is complete.
3. Adds the gravityflowchecklists_post_add_exemption and gravityflowchecklists_post_remove_exemption actions

Fixes [HS5863](https://secure.helpscout.net/conversation/568350537/5863?folderId=1113498)

**Testing Instructions**
For 1:
To reproduce on master, mark one of the forms in a checklist as complete and submit the others. The checklist will not appear as complete. After switching to this branch, the checklist will appear as complete.
Test also be marking all forms as exempt.

For 2:
This is more of change of behaviour than a fix although it could also be considered a fix. On master, a checklist will appear as complete when the last form in the list has a workflow in progress. On this branch the checklist will not appear complete until the workflow on the last form is complete.
